Pipfile

```diff
@@ -5,6 +5,9 @@ name = "pypi"
 
 [packages]
 "discord.py" = "*"
+"pytz" = "*"
+"python-dotenv" = "*"
```

Comment:
Why are you installing dotenv?

Reply:
> Why are you installing dotenv?

I was using it initially but it can be removed I think